.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+Thanks for stopping by to let us know something could be better!
+
+**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.
+
+Please run down the following list and make sure you've tried the usual "quick fixes":
+
+  - Search the issues already opened: https://github.com/googleapis/google-http-client/issues
+  - Check for answers on StackOverflow: http://stackoverflow.com/questions/tagged/google-cloud-platform
+
+If you are still having issues, please be sure to include as much information as possible:
+
+#### Environment details
+
+1. Specify the API at the beginning of the title (for example, "BigQuery: ...")
+   General, Core, and Other are also allowed as types
+2. OS type and version:
+3. Java version:
+4. google-http-client version(s):
+
+#### Steps to reproduce
+
+  1. ?
+  2. ?
+
+#### Code example
+
+```java
+// example
+```
+
+#### Stack trace
+```
+Any relevant stacktrace here.
+```
+
+#### External references such as API reference guides used
+
+- ?
+
+#### Any additional information below
+
+
+Making sure to follow these steps will guarantee the quickest resolution possible.
```

Comment:
"Making sure to follow" --> Following
will guarantee --> guarantees

and perhaps "guarantee" is overly strong